087-adopt-connect-health-endpoint.md

```diff
@@ -0,0 +1,87 @@
+# Adopt the Kafka Connect health check endpoint
+
+In a Kubernetes node, the kubelet component uses the configured liveness probe to know when to restart a container (i.e. deadlock).
```

Comment:
```suggestion
In a Kubernetes node, the kubelet component uses the configured liveness probe to know when to restart a container. 
With a suitably written application this can be used to restart a container if the applications process enters an illegal state (e.g. deadlock).
```

Reply:
Thanks Tom.